sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/ProgressReporter.scala

```diff
@@ -178,7 +178,7 @@ trait ProgressReporter extends Logging {
       id = id,
       runId = runId,
       name = name,
-      timestamp = formatTimestamp(currentTriggerStartTimestamp),
+      timestamp = formatTimestamp(currentTriggerStartTimestamp / NANOS_PER_MILLIS),
```

Comment:
@HeartSaVioR Fixed the timestamp in `StreamingQueryProgress`

Reply:
Looks like you didn't get what @srowen pointed out. `nanoTime` is not a wall-clock - it is only effective when you calculate the difference between twos to get elapsed time.

https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#nanoTime--

Reply:
Yes, this is the point. We do want wall-clock time if the value 'escapes' the JVM, as it does here.

Reply:
get it, thanks